ccc-js/src/main/javascript/package-res/ccc/core/cartesian/axis/cart-axis.js

```diff
@@ -416,6 +416,7 @@ def('pvc.visual.CartesianAxis', pvc_Axis.extend({
         calcContinuousTicks: function(tickCountMax) {
             return this.scale.ticks(this.desiredTickCount(), {
                 roundInside:  this.option('DomainRoundMode') !== 'tick',
+                preserveLastTick: this.option('TicksRegularity'),
```

Comment:
We should be capturing the "tick to use for alignment" (something like `alignmentValue`) in the CCC side, from the axis state of the previous successful render, and then passing it to the protovis ticks method. Any one of the tick values from the previous render's ticks will do.